src/v/kafka/client/client.cc

```diff
@@ -52,14 +53,57 @@ client::client(const YAML::Node& cfg)
                   return mitigate_error(std::move(ex));
               }} {}
 
+ss::future<> client::do_authenticate(shared_broker_t broker) {
+    if (_config.sasl_mechanism().empty()) {
+        vlog(kclog.debug, "Connecting to broker without authentication");
```

Comment:
Nit: There's no equivalent log for when authentication is enabled.

Reply:
fixed